### PR TITLE
DPC-554 - add check to make sure Accept header is set to application/fhir+json

### PR DESCRIPTION
**Pull request by richbraman**
… Accept  header is not null or empty string, and a check to make sure its is set to application/fhir+json

### Fixes [DPC-554](https://jira.cms.gov/browse/DPC-554)

FHIR Bulk data spec requires the Accept parameter to be set to application/fhir+json

### Proposed Changes
add check to make sure Accept header is set to application/fhir+json
Also added swagger annotation to make sure Accept header is required and set to Application/fhir+json 

### Change Details
no further details.

### Security Implications

No; n/A
- [ ] new software dependencies
None

- [ ] security controls or supporting software altered
No

- [ ] new data stored or transmitted
No

- [ ] security checklist is completed for this change
n/A

- [ ] requires more information or team discussion to evaluate security implications
No

- [ X] no PHI/PII is affected by this change

### Acceptance Validation

Yes, via postman

### Feedback Requested

feedback on annotations

### CODE REVIEW

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java [hunk 1]
```diff
@@ -351,18 +355,24 @@ private OffsetDateTime handleSinceQueryParam(String since) {
      *
      * @param outputFormat param to check
      */
-    private static void checkExportRequest(String outputFormat, String headerPrefer) {
-        // _outputFormat only supports FHIR_NDJSON
+    private static void checkExportRequest(String outputFormat, String, Prefer String headerAccept) {
```
Comment (dhgreene) [inline dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java:358]:
```suggestion
    private static void checkExportRequest(String outputFormat, String headerPrefer, String headerAccept) {
```

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java [hunk 2]
```diff
@@ -266,11 +269,12 @@ public Response export(@ApiParam(hidden = true)
                            @QueryParam("_outputFormat") @NoHtml String outputFormat,
                            @ApiParam(value = "Resources will be included in the response if their state has changed after the supplied time (e.g. if Resource.meta.lastUpdated is later than the supplied _since time).")
                            @QueryParam("_since") @NoHtml String since,
-                           @HeaderParam("Prefer")  @Valid String Prefer) {
+                           @HeaderParam("Prefer")  @Valid String Prefer,
+                           @HeaderParam("Accept") @Valid String Accept) {
```
Comment (dhgreene) [inline dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java:273]: Let's change these two parameters (`Prefer` and `Accept`) to lowercase

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java [hunk 3]
```diff
@@ -351,18 +355,24 @@ private OffsetDateTime handleSinceQueryParam(String since) {
...
+      // _outputFormat only supports FHIR_NDJSON
         if (StringUtils.isNotEmpty(outputFormat) && !FHIR_NDJSON.equals(outputFormat)) {
             throw new BadRequestException("'_outputFormat' query parameter must be 'application/fhir+ndjson'");
         }
+        if (headerAccept==null || StringUtils.isEmpty(headerAccept)){
+            throw new BadRequestException("'Accept' header must be 'application/fhir+json'");
+        }
+        if (StringUtils.isNotEmpty(headerAccept) && !FHIR_JSON.equals(headerAccept)) {
+            throw new BadRequestException("'Accept' header must be 'application/fhir+json'");
+        }
```
Comment (dhgreene) [inline dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java:369]:
```suggestion
        if (StringUtils.isEmpty(headerAccept) || !FHIR_JSON.equals(headerAccept)){
            throw new BadRequestException("'Accept' header must be 'application/fhir+json'");
        }
```
This should also apply to the `headerPrefer` if statements below

**Comment by jonfulk** [comment]
Closing this PR in favor of this new one: https://github.com/CMSgov/dpc-app/pull/1091

**Pull request closed without merging**